build/android/BUILD.gn

```diff
@@ -0,0 +1,340 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
```

Comment:
I don't see why this file was added here. `build/android` contains helper scripts used during the build, there's nothing to be built _from_ here.